make/modules/java.base/Copy.gmk

```diff
@@ -277,3 +277,18 @@ $(eval $(call SetupTextFileProcessing, CREATE_CLASSFILE_CONSTANTS_H, \
 TARGETS += $(CREATE_CLASSFILE_CONSTANTS_H)
 
 ################################################################################
+
```

Comment:
Please remove lines 280 and 281. Also add a # SapMachine annotation in shared code.

Reply:
Done